odyssey/client/src/Authentication/Authentication.svelte

```diff
@@ -44,8 +45,9 @@
 	<div class='form'>
 		
 		<form autocomplete="off">
-
-			{#if login}
+			{#if registered}
+				<p>Grumni se ognqne! :)</p>
```

Comment:
Please add a proper text so that the user knows that they have to verify their email.

Reply:
Sorry for that :smiley: 
The new message is "Thank you for registering, you'll receive a validation mail shortly."

Reply:
Good job! 😎